### PR TITLE
Updated to 1.5 (Dev Build)

### PR DESCRIPTION
**Pull request by tylerhasman**
Updated all imports to 1.5 (Dev Build) Tested and works :D

Welcome :D

### CODE REVIEW

File: src/com/lenis0012/bukkit/btm/BTMTaskManager.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public void run() {
 					render.put(name, list);
 				}
 			}
-		}, 20, 20);
+		}, 20, 100);
```
Comment (lenis0012) [inline src/com/lenis0012/bukkit/btm/BTMTaskManager.java:72]: Why did you change the delay here?

**Comment by lenis0012** [comment]
Your version will not work on 1.5 when you dont use ProtocolLib.
I did my own commit with soem of your changes in it.
Thank you though

But in mc 1.5 the field 'd' in ServerConnection has changed to 'c'
We rely on this in the PacketConnection class.

I will also be assding your listener changes because i forgot about that ;)
Good work!

File: src/com/lenis0012/bukkit/btm/BTMTaskManager.java (hunk 1 above)
Comment (tylerhasman) [inline src/com/lenis0012/bukkit/btm/BTMTaskManager.java:72]: Woops, I was playing around and I must have forgotten to change that back
